getting-started/windows-calico/quickstart.md

```diff
@@ -52,6 +53,44 @@ This is required to prevent Linux nodes from borrowing IP addresses from Windows
 calicoctl ipam configure --strictaffinity=true
 ```
 
+#### Prepare {{site.prodname}} networking
+
+Follow the steps below to prepare {{site.prodname}} networking.
+
+**VXLAN networking** 
+
+Disable BGP. 
+```bash
+kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
+```
+**BGP networking** 
+
+Enable BGP service on Windows node. 
+Install the RemoteAccess service using the following Powershell commands:
+
+```powershell
+PS C:\> Install-WindowsFeature RemoteAccess
+PS C:\> Install-WindowsFeature RSAT-RemoteAccess-PowerShell
+PS C:\> Install-WindowsFeature Routing
+```
+
+Then restart the computer:
+
+```powershell
+PS C:\> Restart-Computer -Force
+```
+
+before running:
+
+```powershell
+PS C:\> Install-RemoteAccess -VpnType RoutingOnly
+```
+Sometimes the remote access service fails to start automatically after install. To make sure it is running, execute the following command:
+
+```powershell
+PS C:\> Start-Service RemoteAccess
+```
+
 #### Install {{site.prodnameWindows}}
 
 The following steps install a Kubernetes cluster on a single Windows node, with a Linux control node.
```

Comment:
The geeky details of what you get for kubernetes is wrong - it will now depend on what networking you choose